cli/ops.rs

```diff
@@ -815,17 +815,24 @@ fn op_mkdir(
 ) -> Box<OpWithError> {
   assert!(data.is_none());
   let inner = base.inner_as_mkdir().unwrap();
-  let path = String::from(inner.path().unwrap());
+  let path = match deno_dir::resolve_file_url(
+    inner.path().unwrap().to_string(),
+    ".".to_string(),
+  ) {
+    Err(err) => return odd_future(DenoError::from(err)),
+    Ok(v) => v.to_file_path().unwrap(),
+  };
+  let path_ = path.to_str().unwrap().to_string();
```

Comment:
It seems this bit of code repeated a lot in your additions. Is it possible to abstract/simplify this into a separate function?